### PR TITLE
Step 7 Complete

### PR DESCRIPTION
**Pull request by ggr1992**

### CODE REVIEW

**Review by MrMcSnail — CHANGES_REQUESTED** [review]
Good work so far! 

The votes for your reviews are not persistent between sessions because you've not made an update to the database. 

Once you've got that working, push your changes to this branch and then resubmit an nchelp pr

File: src/Components/SingleReview.jsx [hunk 1]
```diff
@@ -29,11 +29,20 @@ function SingleReview (props) {
...
     };
     
-  const updateVote = (increment) => {
-    setVoteCount((prevCount) => prevCount + (increment ? 1 : -1));
+  const updateVoteUp = () => {
+    setVoteCount((prevCount) => prevCount +  1 );
+    patchVotesIncrease(review_id).then((votesCount) => {
+      setVoteCount(votesCount)
+  })
   };
```
Comment (MKD01) [inline src/Components/SingleReview.jsx]: You shouldnt update votes with the new value from the patch request if you are optimistically updating this value already.

File: src/Components/SingleReview.jsx [hunk 2]
```diff
@@ -29,11 +29,20 @@ function SingleReview (props) {
...
-
```
Comment (MKD01) [inline src/Components/SingleReview.jsx]: If this patch request fails for any reason the user should be given some form of visual feedback.

**Review by MKD01 — CHANGES_REQUESTED** [review]
great work so far but there are still a few things to take a look at before merging this in. 

Once you have made those changes please re-submit this as a new pr.

**Review by Jodiech — APPROVED** [review]
Great work! Your votes are now persistent on a refresh 🥳 Happy for you to merge 🚀

**Pull request merged**